src/d3a/d3a_core/redis_connections/redis_communication.py

```diff
@@ -244,4 +243,4 @@ def heartbeat_tick(self):
 
 def publish_job_error_output(job_id, traceback):
     StrictRedis.from_url(REDIS_URL).\
-        publish(ERROR_CHANNEL, json.dumps({"job_id": job_id, "errors": traceback}))
+        publish(f"{ERROR_CHANNEL}/{job_id}", json.dumps({"errors": traceback}))
```

Comment:
This will crash on the d3a-web/results service side. Better to revert to the old behavior